docs/get-started/deploy-cloudfoundry-for-enterprise.md

```diff
@@ -0,0 +1,129 @@
+# Deploy Cloud Foundry using multiple storage accounts and availability sets
+
+This document provides additional configuration options for high availability and performance of your Cloud Foundry infrastructure.
+For basic Cloud Foundry deployment steps, click [**HERE**](./deploy-cloudfoundry.md).
+ 
+## Availability Set
+
+### Why
+
+For high availability of your Cloud Foundry infrastructure, you can utilize Availability Set for each of your resource pools.
+
+There are two types of Microsoft Azure platform events that can affect the availability of your virtual machines: planned maintenance and unplanned maintenance. To reduce the impact of downtime due to one or more of these events, we recommend configuring multiple virtual machines in an [Availability Set](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-manage-availability/) for redundancy.
+
+### How
+
+1. Open your manifest, and configure an availability set into your `resource_pools`.
+
+  ```
+  - name: resource_router
+    network: default
+    stemcell:
+      name: bosh-azure-hyperv-ubuntu-trusty-go_agent
+      version: '0000'
+    cloud_properties:
+      availability_set: <availability-set-name>
+      instance_type: Standard_D1
+  ```
+
+  If `<availability-set-name>` does not exist, it will be created automatically.
+
+2. Specify the resource_pool as `resource_router` in your job. Then these 2 VMs will be put into the availability set `<availability-set-name>`.
+
+  ```
+  - name: router_z1
+    instances: 2
+    resource_pool: resource_router
+    templates:
+    - {name: gorouter, release: cf}
+    - {name: metron_agent, release: cf}
+    networks:
+    - name: default
+      static_ips: [10.0.16.12, 10.0.16.22]
+    properties:
+      dropsonde: {enabled: true}
+  ```
+
+3. Deploy your Cloud Foundry.
+
+## Multiple Storage Account
+
+### Why
+
```

Comment:
Here is the suggestions for paragraph "Why"
 By default there is one Standard Tier storage account created for each resource pool, providing 40 IOPS per disk based on Azure storage design(insert link here). If you need higher performance, you need to create multiple storage accounts. In addition, since each resource pool can only contain single storage account, with multiple storage accounts, you also need to split the resource pool into multiple resource pools, each with a single storage account. 
 Note you can choose different types of storage accounts when considering performance optimization. For example, if you need high-performance, low-latency disk support, you can also use Azure Premium Storage, please keep in mind the maximum number of highly utilized disks per storage account is about 31 for Premium Storage.